benefit/data/sheets/FixedLists/FixedList-sandl-nonprod.json

```diff
@@ -2,5 +2,6 @@
   {"LiveFrom": "06/07/2022", "ID": "FL_amendReason", "ListElementCode": "partyreq", "ListElement": "Party requested change"},
   {"LiveFrom": "06/07/2022", "ID": "FL_amendReason", "ListElementCode": "judgereq", "ListElement": "Judge requested change"},
   {"LiveFrom": "06/07/2022", "ID": "FL_amendReason", "ListElementCode": "adminreq", "ListElement": "Admin requested change"},
-  {"LiveFrom": "06/07/2022", "ID": "FL_amendReason", "ListElementCode": "adminerr", "ListElement": "Admin error"}
+  {"LiveFrom": "06/07/2022", "ID": "FL_amendReason", "ListElementCode": "adminerr", "ListElement": "Admin error"},
+  {"LiveFrom": "12/08/2022", "ID": "FL_dwpStates", "ListElementCode": "hearingDateIssued", "ListElement": "Hearing Date Issued"}
```

Comment:
Don't add a field to both non-prod and prod sheets, it will cause an error. If you want something behind a feature flag and not to go to production, add it to a nonprod sheet.